config/data/canonical_states.yml

```diff
@@ -0,0 +1,47 @@
+India:
+  - Andhra Pradesh
+  - Arunachal Pradesh
+  - Assam
+  - Bihar
+  - Chhattisgarh
+  - Goa
+  - Gujarat
+  - Haryana
+  - Himachal Pradesh
+  - Jharkhand
+  - Karnataka
+  - Kerala
+  - Madhya Pradesh
+  - Maharashtra
+  - Manipur
+  - Meghalaya
+  - Mizoram
+  - Nagaland
+  - Odisha
+  - Punjab
+  - Rajasthan
+  - Sikkim
+  - Tamil Nadu
+  - Telangana
+  - Tripura
+  - Uttar Pradesh
+  - Uttarakhand
+  - West Bengal
+Bangladesh:
+  - Barisal
+  - Bogura
+  - Chittagong
+  - Comilla
+  - Dhaka North
+  - Dhaka South
+  - Gazipur
+  - Khulna
+  - Mymensingh
+  - Narayanganj
+  - Rajshahi
+  - Rangpur
+  - Sylhet
+Ethiopia:
+  - አዲስ አበባ
+  - ትግራይ
+  - Oromiyaa
```

Comment:
This is not the complete list of Ethiopian states, we can update this in a later PR.